fmriprep/interfaces/bids.py

```diff
@@ -162,6 +162,8 @@ class DerivativesDataSinkInputSpec(BaseInterfaceInputSpec):
     source_file = File(exists=False, mandatory=True, desc='the input func file')
     suffix = traits.Str('', mandatory=True, desc='suffix appended to source_file')
     extra_values = traits.List(traits.Str)
+    force_uncompress = traits.Bool(False, usedefault=True,
+                                   desc="save as uncompressed nifti")
```

Comment:
I might just do this as

```Python
compress = traits.Bool(desc="output file as compressed (True) or uncompressed (False) (default: unchanged from input)")
```

Or something similar. That seems more natural to me.